core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -191,7 +191,7 @@
       + "server-internal scheduled tasks"),
   // If you update the default type, be sure to update the default used for initialization failures in VolumeManagerImpl
   @Experimental
-  GENERAL_VOLUME_CHOOSER("general.volume.chooser", "org.apache.accumulo.server.fs.PerTableVolumeChooser", PropertyType.CLASSNAME,
+  GENERAL_VOLUME_CHOOSER("general.volume.chooser", "org.apache.accumulo.server.fs.RandomVolumeChooser", PropertyType.CLASSNAME,
```

Comment:
I'm actually questioning whether this is necessary myself. The original intent of the parent issue was to ban certain volumes from being used entirely (unless explicitly desired). This change was to help provide a sane fall-back scenario, but I'm not sure it's necessary if we change things to be a hard-fail in the case of misconfiguration.